lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -375,8 +375,7 @@ sub to_yaml {
     }
     my $hash = $self->to_template;
 
-    # Note: Stripping the initial document start marker "---"
-    my $yaml = YAML::XS::Dump($hash) =~ s/^---\n//r;
+    my $yaml = dump_yaml(string => $hash);
```

Comment:
Better just return it and avoid the extra local variable (and the same in `expand_yaml`).